lab_2_tokenize_by_bpe/main.py

```diff
@@ -5,7 +5,7 @@
 
 
 def prepare_word(
-    raw_word: str, start_of_word: str | None, end_of_word: str | None
+        raw_word: str, start_of_word: str | None, end_of_word: str | None
```

Comment:
Function definitions should stay the same 🏄

Reply:
fixed

Reply:
doesn't look like it

Reply:
fixed?